Active_Record/.idea/.rakeTasks

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
remove folder `.idea` from a repo and add this folder to a `.gitignore`

Reply:
done